.editorconfig

```diff
@@ -2,8 +2,19 @@
 root = true
 
 [*]
+insert_final_newline = true
+charset = utf-8
+
+[*.sln]
+end_of_line = crlf
+indent_style = tab
+
+[*.csproj]
+end_of_line = crlf
```

Comment:
Pls for the love of all that's hoyl can we yeet the CRLF and just convert everything to LF to be consistent?

Reply:
yep, sure. i see no reason to stick with this inconsistency. was just making the .gitignore match our docs

Reply:
The only reason for the inconsistency was to match VS's shitty defaults. Seeing as VS follows editorconfig I'm all for making things consistent.